.github/workflows/rust.yml

```diff
@@ -92,27 +82,53 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
-      - name:                   Checkout repository
-        uses:                   actions/checkout@v2
-      - uses: actions/cache@v3
+      - uses:                   actions/checkout@v2
+      - run: rustup toolchain install stable --profile minimal
+      - uses: Swatinem/rust-cache@v2
         with:
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          key: "coverage"
       - name: Setup database
         run: |
           cargo install sqlx-cli --no-default-features --features native-tls,postgres
           cargo sqlx database setup
       - name:                   Generate code coverage
         run: |
-          cargo +nightly tarpaulin --verbose --all-features --workspace --timeout 120 --out Xml
+          cargo tarpaulin --all-targets --skip-clean --workspace --timeout 120 --out Xml
       - name:                   Upload to codecov.io
         uses:                   codecov/codecov-action@v2
         with:
           token:                ${{secrets.CODECOV_TOKEN}}
           fail_ci_if_error:     true
 
+
+  test:
```

Comment:
we now either need to specify that tests must also pass before merging is allowed (which must be done by an admin somewhere in the repo) or we maybe can create a dummy task which just depends on all tasks that need to be successfull before merging is allowed (needs see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds)

Reply:
You're the admin....

Reply:
And i would like to have only one job which does nothing and depends on many children doing the work in parallel (if that works like i imagine)